windows/wsl_distro_install_uninstall/install_wsl.yml

```diff
@@ -9,25 +9,58 @@
 # and the support of inbox version is removed from Windows build 25267, the test case will install inbox version for
 # build lower than 25267 until the known issue is resolved.
 #
-- name: "Set installation command for inbox version of WSL"
+- name: "Initialize some facts"
   ansible.builtin.set_fact:
-    wsl_install_cmd: "wsl --install --inbox -d {{ wsl_distribution_name }} | Out-File -FilePath {{ wsl_file_path_win }}\\wsl_install.txt"
-  when:
-    - guest_os_build_num | int < 25267
-    - guest_os_build_num | int > 22000
+    skip_set_wsl_ver: false
 
-- name: "Set installation command for store version of WSL"
-  ansible.builtin.set_fact:
-    wsl_install_cmd: "wsl --install -d {{ wsl_distribution_name }} | Out-File -FilePath {{ wsl_file_path_win }}\\wsl_install.txt"
-  when: > 
-    (guest_os_build_num | int >= 25267) or
-    (guest_os_build_num | int < 22000)
+- name: "Install WSL for Windows Client"
+  block:
+    - name: "Set installation command for inbox version of WSL"
+      ansible.builtin.set_fact:
+        wsl_install_cmd: "wsl --install --inbox -d {{ wsl_distribution_name }} | Out-File -FilePath {{ wsl_file_path_win }}\\wsl_install.txt"
+      when:
+        - guest_os_build_num | int < 25267
+        - guest_os_build_num | int > 22000
```

Comment:
How about guest_os_build_num | int == 22000 ?

Reply:
Yes, missing the process for 22000. Fixed. Thanks.